docs/install-pmm/install-pmm-server/baremetal/docker/index.md

```diff
@@ -1,21 +1,24 @@
 # Install PMM Server with Docker container
 
-
 This section provides instructions for running PMM Server with Docker based on the [PMM Docker image](https://hub.docker.com/r/percona/pmm-server).
 
 ## Running PMM Server with Watchtower
 
-To enable PMM Server upgrades via the **Upgrade Now** button on the PMM Home dashboard, you need to set up Watchtower alongside PMM Server during installation. Watchtower is a container monitoring tool that updates Docker containers to the latest version when triggered.
+To enable PMM Server upgrades via the **Upgrade page** and the **Upgrade Now** button on the Home dashboard, you must configure Watchtower during the PMM Server installation. Watchtower is a container monitoring tool that automatically updates Docker containers to their latest version when triggered.
```

Comment:
```suggestion
To enable PMM Server upgrades via the **Upgrade page** and the **Upgrade Now** button on the Home dashboard, you must configure Watchtower during the PMM Server installation. Watchtower is a container monitoring tool that helps update Docker containers to their latest version when triggered.
```

Reply:
IMO the mention of "automatically" could lead the user to think it would happen "without human interaction" even though we say "when triggered".